sdk/core/System.Net.ClientModel/src/System.Net.ClientModel.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains building blocks for clients that call cloud services.</Description>
+    <Version>1.0.0-beta.1</Version>
+    <Nullable>enable</Nullable>
+
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+
+    <!-- TODO: Remove warning suppression -->
+    <NoWarn>$(NoWarn);AZC0001;AZC0012;AZC0014;CS1591;NETSDK1138</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" />
+    <PackageReference Include="System.Memory.Data" />
+    <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.Text.Encodings.Web" />
```

Comment:
Is this needed for model reader writer types?

Reply:
It was there before so I left I will try removing it